tests/studies/test_dataservice.py

```diff
@@ -79,8 +79,8 @@ def test_sync_dataservice_studies_deleted(db, mocker):
     req_mock = mocker.patch("creator.studies.dataservice.requests")
     req_mock.get().json.return_value = MOCK_RESP
 
-    sync_dataservice_studies()
-    assert Study.objects.count() == 2
+    study_1 = StudyFactory(kf_id="SD_DYPMEHHF")
```

Comment:
I don't see these used?

Reply:
They are referred to in the mocked response from the dataservice. `SD_DYPMEHHF` is not there so we make sure it gets marked as deleted. `SD_9PYZAHHE` is there so we make sure it's not deleted.